src/components/Header/Header.jsx

```diff
@@ -101,6 +111,15 @@ class Header extends Component {
             </nav>
           </div>
         </div>
+        {this.state.visiblePrivacyPolicyBar ?
+          <div className="notification">
+            <div className="content">
+              <img className="cookieImg" src={cookieImg} onClick={this.closePrivacyPolicyBar} />
+              <p className="privacyPolicyText">By accepting our usage of third-party software such as HotJar, you help us to deliver a better website experience to all our users. To see our full privacy policy, <a href="https://getsaleor.com/privacy-policy-terms-and-conditions/" target="_blank">click here.</a></p>
```

Comment:
- It doesn't seems to invite the user to accept third-party tools, it doesn't sound as a prompt;
- I would also add that technical words, especially services names (HotJar) should be avoided to target a wider range of users. Maybe we could replace HotJar with "visitor analytics" or something else that doesn't sound scary.

Maybe something like (with some more rewording):
```suggestion
              <p className="privacyPolicyText">Our website is using third-party tools, such as a visitor analytics tool, that help us to deliver you a better experience. Are you agreeing?<a href="https://getsaleor.com/privacy-policy-terms-and-conditions/" target="_blank">More details here</a>.</p>
```

Also, will those services (e.g. HotJar) be loaded only if the user accepts? Or the user as to manually opt-out? If so, we probably should have a "Dismiss" button instead of "Accept", that would make more sense (and thus, we would not need to ask the user "Are you agreeing?").